src/components/atoms/Description/description.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import "./style.scss"
+
+const Description = (props) => {
```

Comment:
Why did you make this a separate component if this component only renders something simple?